src/apiCalls.js

```diff
@@ -0,0 +1,4 @@
+export const getAllStories = () => {
+  return fetch('https://api.nytimes.com/svc/topstories/v2/climate.json?api-key=b3M1MC9DPZ6AYoCBVQ98cGQZYXRjwuoZ')
+    .then(response => response.json())
```

Comment:
Do you want to check for if response.ok is true here? Maybe create an error component?